### DISCUSSION TITLE
Start a truck from a specific time

### DISCUSSION THREAD

**Discussion by alono** [lhz516/react-h5-audio-player#92] [General]
Hi,
I am trying to start playing a src ogg file, but I want to set the start time to play not from the very beginning, but to start from another time provided. 
let's say there is a sound truck of 5.23 minutes, and I want to start playing it from 1.05. 
1. Is it possible? 
2. If yes, How can it be done? 
Thanks

**Comment by lhz516** [comment]
Yes it's possible. Just get the audio ref, and set `audio.currentTime = 65` in your `useEffect` hook or `componentDidMount`.

**Reply by alono** [reply]
Great! thanks.
